lib/bot.js

```diff
@@ -118,6 +119,13 @@ Bot.prototype.parseText = function(text) {
         return '<' + label + '>';
       }
       return '<' + command + '>';
+    })
+    .replace(/\:(\w+)\:/g, function(match) {
+      var clean = match.replace(/\:/g,'');
```

Comment:
Isn't what you're trying to get from `clean` here already passed as the second argument? Since `(\w+)` is a regex capture group. See https://github.com/andebor/slack-irc/blob/master/lib/bot.js#L109.